src/main/java/uk/gov/hmcts/darts/event/service/impl/EventServiceImpl.java

```diff
@@ -26,4 +28,15 @@ public AdminGetEventForIdResponseResult adminGetEventById(Integer eventId) {
         Optional<EventEntity> eventEntityOptional = eventRepository.findById(eventId);
         return eventMapper.mapToAdminGetEventsResponseForId(eventEntityOptional);
     }
+
+    @Override
+    public EventEntity getEventById(Integer eventId) {
```

Comment:
This looks very similar to the validate method in EventIdValidator. Wonder if they can be combined? If not, don't worry too much.

Reply:
Agreed will combine for performance benefits